ibm_ace/tests/agent_scripts/start_commands.sh

```diff
@@ -0,0 +1,17 @@
+# This script makes the necessary setup to be able to compile pymqi on the agent machine
+
+MQ_URL=https://ddintegrations.blob.core.windows.net/ibm-mq/9.1.0.4-IBM-MQC-Redist-LinuxX64.tar.gz
```

Comment:
Not sure if IBM MQ version needs to be consistent, but should this be the same as 9.2.4.0-r1? From https://github.com/DataDog/integrations-core/pull/11654/files?diff=unified&w=0#diff-020f64920c36825420a64d0c4189fed9190d53b92bd4f43bc6c5249b4186e4b6R27